minifi-toolkit/minifi-toolkit-configuration/src/test/resources/1.5_RPG_Handling.yml

```diff
@@ -112,3 +112,5 @@ Remote Process Groups:
     use compression: false
   Output Ports: []
 NiFi Properties Overrides: {}
+Reporting Tasks: []
```

Comment:
Seems this was duplicated.  I also think it should be okay to not have a Reporting Tasks section from a config standpoint.  For existing flows, we do not wish to break them.

Reply:
Agree, I reverted the changes